templates/footer.html

```diff
@@ -7,23 +7,23 @@ <h2 class="p-heading--5">Charmed Operator Framework</h2>
 
       <ul class="p-list">
         <li class="p-list__item">
-          <a class="p-link--soft" href="https://juju.is"><small>What is Juju?</small></a>
+          <a class="p-link--soft" href="/"><small>What is Juju?</small></a>
```

Comment:
Probably want an id to anchor to the what is Juju section?

Reply:
Done